docs/reference/admin/layouts/overview.mdx

```diff
@@ -1,52 +1,28 @@
 ---
-title: Layout overview
+title: Overview
 ---
 
-# <span className="version">Interface 1.2+</span> Layout overview
+import CodeBlock from '@theme/CodeBlock'
+import _pages_exampleSource from '!!raw-loader!./_Example/pages/example.tsx'
+import LayoutKitSvg from '@site/static/assets/layout-kit.svg'
 
-Contember comes with flexible and declarative layout controls to render content in any layout you want.
-And control layout from within any rendered page — introducing **Slots** and **Directives**.
+# <span className="version">Interface 1.2+</span> Layout Kit overview
 
-:::note Slots and Directives are available in Contember Interface version 1.2.
+:::note `LayoutKit`, `Slots` and `Directives` are available in Contember Interface version 1.2 and are meant to be used together with [`Scopes`](/reference/admin/pages/scopes).
```

Comment:
this should be in a content of a note, not in a title

Reply:
Done